packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -35,6 +36,11 @@ export const buttonVariantValues = [
   "floating",
 ] as const;
 
+type NativeButtonProps = Pick<
+  ComponentPropsWithoutRef<"button">,
+  "aria-controls" | "aria-haspopup" | "aria-expanded"
+>;
```

Comment:
I understand why you added this, but why not add the aria props we've been using in other components such as `ariaControls`, `ariaHasPopup`, and `areaExpanded`?

---

Can we also make sure to export this type?
```suggestion
export type NativeButtonProps = Pick<
  ComponentPropsWithoutRef<"button">,
  "aria-controls" | "aria-haspopup" | "aria-expanded"
>;
```